include/macro.inc

```diff
@@ -37,3 +37,39 @@
 .set TagHi,         $29
 .set ErrorEPC,      $30
 .set Reserved31,    $31
+
+
+# Float register aliases (o32 ABI, odd ones are rarely used)
+
+.set $fv0,          $f0
+.set $fv0f,         $f1
+.set $fv1,          $f2
+.set $fv1f,         $f3
+.set $ft0,          $f4
+.set $ft0f,         $f5
+.set $ft1,          $f6
+.set $ft1f,         $f7
+.set $ft2,          $f8
+.set $ft2f,         $f9
+.set $ft3,          $f10
+.set $ft3f,         $f11
+.set $fa0,          $f12
+.set $fa0f,         $f13
+.set $fa1,          $f14
+.set $fa1f,         $f15
+.set $ft4,          $f16
+.set $ft4f,         $f17
+.set $ft5,          $f18
+.set $ft5f,         $f19
+.set $fs0,          $f20
+.set $fs0f,         $f21
+.set $fs1,          $f22
+.set $fs1f,         $f23
+.set $fs2,          $f24
+.set $fs2f,         $f25
+.set $fs3,          $f26
+.set $fs3f,         $f27
+.set $fs4,          $f28
+.set $fs4f,         $f29
+.set $fs5,          $f30
+.set $fs5f,         $f31
```

Comment:
Isn't this the FpcCsr register? Shouldn't it be named like that instead?

Reply:
The FpcCsr register actually belongs to a separate set of registers, the floating-point "control" registers. There are two such registers, $1 the floating-point unit version register, and $31 which is the well-known FpcCsr. The disassembler doesn't implement these properly since $f31/$fs5f is never used and neither is $1, the version register, so I cut corners to avoid adding another operand type. I'm not sure if this is now an issue with this additional machinery being added to the float registers.

Reply:
It's still not implemented in the disassembler, so it doesn't really matter, I don't think... $f31 is not the same as $31 anyway, right?

Reply:
If I remember correctly, the point is that different float instructions treat `$31` in different ways, and it so happens that it's never used by instructions that would use register 31 as `$f31`/`$fs5f`.